scheduler-jobs/deploy_data_specific_image

```diff
@@ -82,10 +82,14 @@ for ((i=0;i<$CONFIG_LIST_LENGTH;i++)); do
 
     echo "Extracting ${FULL_URL} to ${FULL_DEST_DIR}"
 
-    # Download the data from github and copy it to the destination directory
-    # TODO: Handle renamed or deleted notebooks, since this command only copies the content of the source_dir in the dest_dir
-    # For now, any other file found in the dest_dir are not handled and remain in its location.
-    svn export --force $FULL_URL $FULL_DEST_DIR
+    # Download the data from github into tmp directory
+    svn export --force $FULL_URL tmp
+
+    # Synchronize source_dir with dest_dir
+    # --delete : delete extraneous files from destination dirs
+    rsync -rtovcs --delete tmp/ $FULL_DEST_DIR/
```

Comment:
In scripts, it's usually preferable to use the long form of options so it's clearer to the reader.  The short form is useful only to save time when typing manually.  Please use the long options of `rsync`.

By the way, here are the `rsync` options I've used in a very similar script https://github.com/bird-house/birdhouse-deploy/blob/4e9a94dbcbf1ddd8bb275317206c3376fd7897d3/birdhouse/deployment/deploy-data#L154-L156 check if there's any options you're missing that could be useful for this case.

Reply:
Sorry a dumb question but is `rsync` available in the image?  In my script above, I used `rsync` from a separate docker image because I wanted the script to be generic and be able to run inside any images with docker binary.  You case might not have the same requirements so it's fine but check that `rsync` is available.

Reply:
You're question is legit, and yes you're right I could use another docker image in which ```rsync``` is installed, but in my opinion it would make the base image not usable as a base image. If this its the base image, then it has to contain what it needs to work from scratch. So in this case I installed ```rsync``` in it.